### ISSUE TITLE
fatal: unsafe repository ('XXXXXX' is owned by someone else)

### ISSUE THREAD

**Issue by gdonoso94** [cpina/github-action-push-to-another-repository#64]
Hi,

I'm encountering an error when running this action to push to another repo:

```
fatal: unsafe repository ('/tmp/tmp.PklpEM' is owned by someone else)

To add an exception for this directory, call:

	git config --global --add safe.directory /tmp/tmp.PklpEM
```

Looks like this is related to [this](https://github.blog/2022-04-12-git-security-vulnerability-announced/).

I have tried already setting the ownership of the containing folder in a previous step in my job but no luck yet.

Is there any workaround?

Thanks in advance

**Comment by bugoverdose** [comment]
I'm currently facing the same issue. 
It worked fine until few days ago, but now I started seeing the same `fatal: unsafe repository ('/tmp/tmp.XXX' is owned by someone else)` exception message.

**Comment by cpina** [comment]
I'll try to reproduce and have a look - thanks for the heads up!

**Comment by cpina** [comment]
I cannot reproduce it yet (I don't have handy two repos with one not owned by me right now to test with) but I've attempted a fix.

Could you try using the `devel` branch of the action?

In  your `.github/workflows` you might have something like this:
```
      - name: Pushes to another repository
        id: push_directory
        uses: cpina/github-action-push-to-another-repository@v1.4.1
```
Change the v1.4.1 (or perhaps you use `main`) to `devel`:
uses: cpina/github-action-push-to-another-repository@devel

And see if it works. Let me know please.

Thank you very much!

**Comment by bugoverdose** [comment]
I've tried it, but nothing changed. 🥲

<img width="587" alt="image" src="https://user-images.githubusercontent.com/73531614/163119510-81064aa2-7e65-4923-b736-0a60be823400.png">

<img width="548" alt="image" src="https://user-images.githubusercontent.com/73531614/163119536-80dd244c-6345-44d5-ab52-eb0d7494173c.png">

**Comment by gdonoso94** [comment]
@cpina used `@devel`. Nothing changed though. 

In case you need more information to reproduce the error: I'm using the `checkout@v2` action to pull the _other_ repository, modifying files on it, and trying to push them back.

**Comment by cpina** [comment]
I'll try to reproduce the problem and fix it again. Thanks for testing and reporting back.

Are any of you using it in public repos that I could have the URL? It might help looking better into it.

Thanks!

**Comment by gdonoso94** [comment]
I'm using it in private repos, sorry 😞 .

**Comment by zoha-im** [comment]
Having the same problem! 😥 was working a few days back! 
![image](https://user-images.githubusercontent.com/67825158/163122308-5e93149e-115f-487d-ac95-6329d866b319.png)

Here's a partial screengrab of the workflow code
![image](https://user-images.githubusercontent.com/67825158/163121840-74156304-030a-410f-9b81-99b75a6c38ce.png)

Tried using cpina/github-action-push-to-another-repository@devel
Tried using cpina/github-action-push-to-another-repository@1.4.1

Both are private repos owned by the organizations and I have ownership privileges.

**Comment by bugoverdose** [comment]
I can show you these info if that will help.

- .github/workflows/master.yml

```yml
name: CI/CD to different repository

on:
  push:
    branches: [master]

jobs:
  deploy:
    runs-on: ubuntu-latest

    steps:
      - name: Checkout blog repository
        uses: actions/checkout@v2

      - name: Setup Node.js on runner
        uses: actions/setup-node@v2
        with:
          node-version: "16.13.2"

      - name: Install dependencies
        run: npm install

      - name: Build
        run: npm run build

      - name: Push to github.io repository
        uses: cpina/github-action-push-to-another-repository@devel
        env:
          API_TOKEN_GITHUB: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
        with:
          source-directory: "public"
          destination-github-username: "bugoverdose"
          destination-repository-name: "bugoverdose.github.io"
          user-email: jwjeong96@gmail.com
          target-branch: master
```

- Github Action (Push to github.io repository)

```
[+] Action start
[+] Git version
git version 2.34.2
[+] Add everything is safe
[+] Cloning destination git repository bugoverdose.github.io
Cloning into '/tmp/tmp.KPHMGb'...
total 5032
drwx------   17 root     root          4096 Apr 13 07:04 .
drwxrwxrwt    1 root     root          4096 Apr 13 07:04 ..
drwxr-xr-x    8 root     root          4096 Apr 13 07:04 .git
-rw-r--r--    1 root     root           805 Apr 13 07:04 175-0dcd6aa6c21079d8aa05.js
-rw-r--r--    1 root     root          3391 Apr 13 07:04 175-0dcd6aa6c21079d8aa05.js.map
-rw-r--r--    1 root     root          3129 Apr 13 07:04 231-4654f5f3ac3fe1e4eaaf.js
-rw-r--r--    1 root     root          8438 Apr 13 07:04 231-4654f5f3ac3fe1e4eaaf.js.map
drwxr-xr-x    2 root     root          4096 Apr 13 07:04 404
-rw-r--r--    1 root     root         11235 Apr 13 07:04 404.html
-rw-r--r--    1 root     root          1216 Apr 13 07:04 67-4639f193183c83596226.js
-rw-r--r--    1 root     root          5717 Apr 13 07:04 67-4639f193183c83596226.js.map
-rw-r--r--    1 root     root        307860 Apr 13 07:04 7112840a-56005de57589deff82e4.js
-rw-r--r--    1 root     root          7781 Apr 13 07:04 7112840a-56005de57589deff82e4.js.LICENSE.txt
-rw-r--r--    1 root     root       1346749 Apr 13 07:04 7112840a-56005de57589deff82e4.js.map
drwxr-xr-x    2 root     root          4096 Apr 13 07:04 7a4f853b6ed1b1d270733af0138e8a65
-rw-r--r--    1 root     root        [14](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:14)6808 Apr 13 07:04 app-61b28d6b40234e1de187.js
-rw-r--r--    1 root     root           487 Apr 13 07:04 app-61b28d6b40234e1de187.js.LICENSE.txt
-rw-r--r--    1 root     root        549553 Apr 13 07:04 app-61b28d6b40234e1de187.js.map
-rw-r--r--    1 root     root         11873 Apr 13 07:04 bugoverdose-logo.png
-rw-r--r--    1 root     root          2457 Apr 13 07:04 bugoverdose-open-graph.png
drwxr-xr-x    2 root     root          4096 Apr 13 07:04 categories
drwxr-xr-x    5 root     root          4096 Apr 13 07:04 category
-rw-r--r--    1 root     root           774 Apr 13 07:04 chunk-map.json
-rw-r--r--    1 root     root        188560 Apr 13 07:04 commons-7dadb1846d649c26545a.js
-rw-r--r--    1 root     root          2423 Apr 13 07:04 commons-7dadb1846d649c26545a.js.LICENSE.txt
-rw-r--r--    1 root     root        700725 Apr 13 07:04 commons-7dadb1846d649c26545a.js.map
-rw-r--r--    1 root     root          6331 Apr 13 07:04 component---src-pages-404-js-9c0125c4edb459993e76.js
-rw-r--r--    1 root     root         [15](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:15)321 Apr 13 07:04 component---src-pages-404-js-9c0125c4edb459993e76.js.map
-rw-r--r--    1 root     root         15807 Apr 13 07:04 component---src-pages-categories-js-7840dc28e8663edac40d.js
-rw-r--r--    1 root     root         3[16](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:16)72 Apr 13 07:04 component---src-pages-categories-js-7840dc28e8663edac40d.js.map
-rw-r--r--    1 root     root         10908 Apr 13 07:04 component---src-pages-mdx-slug-js-2702498268d1bf8327f2.js
-rw-r--r--    1 root     root         3[17](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:17)[18](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:18) Apr 13 07:04 component---src-pages-mdx-slug-js-2702498268d1bf8327f2.js.map
-rw-r--r--    1 root     root         14629 Apr 13 07:04 component---src-pages-tags-js-509e3e8187123b8eef4a.js
-rw-r--r--    1 root     root         34729 Apr 13 07:04 component---src-pages-tags-js-509e3e8187123b8eef4a.js.map
-rw-r--r--    1 root     root         10902 Apr 13 07:04 component---src-templates-category-js-a2d347749b99ab9b64cc.js
-rw-r--r--    1 root     root         30407 Apr 13 07:04 component---src-templates-category-js-a2d347749b99ab9b64cc.js.map
-rw-r--r--    1 root     root         12232 Apr 13 07:04 component---src-templates-home-js-0f7ea95a8bf2f2e01af6.js
-rw-r--r--    1 root     root         31018 Apr 13 07:04 component---src-templates-home-js-0f7ea95a8bf2f2e01af6.js.map
-rw-r--r--    1 root     root         10966 Apr 13 07:04 component---src-templates-tag-js-585d7b1dd54cc9e8a5f6.js
-rw-r--r--    1 root     root         30598 Apr 13 07:04 component---src-templates-tag-js-585d7b1dd54cc9e8a5f6.js.map
drwxr-xr-x    6 root     root          4096 Apr 13 07:04 computer-science
drwxr-xr-x    9 root     root          4096 Apr 13 07:04 development
drwxr-xr-x    3 root     root          4096 Apr 13 07:04 essay
-rw-r--r--    1 root     root           428 Apr 13 07:04 favicon-32x32.png
-rw-r--r--    1 root     root        129753 Apr 13 07:04 framework-d990a81097fc07e0893d.js
-rw-r--r--    1 root     root           736 Apr 13 07:04 framework-d990a81097fc07e0893d.js.LICENSE.txt
-rw-r--r--    1 root     root        3[19](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:19)864 Apr 13 07:04 framework-d990a81097fc07e0893d.js.map
drwxr-xr-x    2 root     root          4096 Apr 13 07:04 icons
-rw-r--r--    1 root     root         11254 Apr 13 07:04 index.html
-rw-r--r--    1 root     root          1005 Apr 13 07:04 manifest.webmanifest
drwxr-xr-x    3 root     root          4096 Apr 13 07:04 page
drwxr-xr-x   14 root     root          4096 Apr 13 07:04 page-data
-rw-r--r--    1 root     root         85840 Apr 13 07:04 polyfill-4b355b7cb1b252846bfb.js
-rw-r--r--    1 root     root        [20](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:20)5337 Apr 13 07:04 polyfill-4b355b7cb1b252846bfb.js.map
-rw-r--r--    1 root     root           124 Apr 13 07:04 robots.txt
-rw-r--r--    1 root     root        612063 Apr 13 07:04 rss.xml
drwxr-xr-x    2 root     root          4096 Apr 13 07:04 sitemap
drwxr-xr-x   52 root     root          4096 Apr 13 07:04 static
-rw-r--r--    1 root     root          2659 Apr 13 07:04 styles.e1fc8df16ade7f667a96.css
drwxr-xr-x   27 root     root          4096 Apr 13 07:04 tag
drwxr-xr-x    2 root     root          4096 Apr 13 07:04 tags
-rw-r--r--    1 root     root          4265 Apr 13 07:04 webpack-runtime-9fa3f628f5389b3d2572.js
-rw-r--r--    1 root     root         18869 Apr 13 07:04 webpack-runtime-9fa3f628f5389b3d2572.js.map
-rw-r--r--    1 root     root          4803 Apr 13 07:04 webpack.stats.json
[+] Deleting /tmp/tmp.KPHMGb//
[+] Creating (now empty) /tmp/tmp.KPHMGb//
[+] Listing Current Directory Location
total 1680
drwxr-xr-x   10 1001     1[21](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:21)           4096 Apr 13 07:03 .
drwxr-xr-x    6 root     root          4096 Apr 13 07:04 ..
drwxr-xr-x   18 1001     121           4096 Apr 13 07:04 .cache
-rw-r--r--    1 1001     121            245 Apr 13 07:01 .env.development
-rw-r--r--    1 1001     121            245 Apr 13 07:01 .env.production
drwxr-xr-x    8 1001     121           4096 Apr 13 07:01 .git
drwxr-xr-x    3 1001     121           4096 Apr 13 07:01 .github
-rw-r--r--    1 1001     121             57 Apr 13 07:01 .gitignore
-rw-r--r--    1 1001     121          10892 Apr 13 07:01 README.md
drwxr-xr-x    2 1001     121           4096 Apr 13 07:01 config
-rw-r--r--    1 1001     121            338 Apr 13 07:01 gatsby-browser.js
-rw-r--r--    1 1001     121           8069 Apr 13 07:01 gatsby-config.js
-rw-r--r--    1 1001     121           1062 Apr 13 07:01 gatsby-node.js
drwxr-xr-x 1176 1001     121          36864 Apr 13 07:03 node_modules
-rw-r--r--    1 1001     121        1600815 Apr 13 07:03 package-lock.json
-rw-r--r--    1 1001     121           1466 Apr 13 07:01 package.json
drwxr-xr-x   16 1001     121           4096 Apr 13 07:04 public
drwxr-xr-x   12 1001     121           4096 Apr 13 07:01 src
drwxr-xr-x    2 1001     121           4096 Apr 13 07:01 static
[+] Listing root Location
total 84
drwxr-xr-x    1 root     root          4096 Apr 13 07:04 .
drwxr-xr-x    1 root     root          4096 Apr 13 07:04 ..
-rwxr-xr-x    1 root     root             0 Apr 13 07:04 .dockerenv
drwxr-xr-x    2 root     root          4096 Apr  4 16:06 bin
drwxr-xr-x    5 root     root           340 Apr 13 07:04 dev
-rwxr-xr-x    1 root     root          4397 Apr 13 06:50 entrypoint.sh
drwxr-xr-x    1 root     root          4096 Apr 13 07:04 etc
drwxr-xr-x    6 root     root          4096 Apr 13 07:04 github
drwxr-xr-x    2 root     root          4096 Apr  4 16:06 home
drwxr-xr-x    1 root     root          4096 Apr  4 16:06 lib
drwxr-xr-x    5 root     root          4096 Apr  4 16:06 media
drwxr-xr-x    2 root     root          4096 Apr  4 16:06 mnt
drwxr-xr-x    2 root     root          4096 Apr  4 16:06 opt
dr-xr-xr-x  213 root     root             0 Apr 13 07:04 proc
drwx------    2 root     root          4096 Apr  4 16:06 root
drwxr-xr-x    1 root     root          4096 Apr 13 07:04 run
drwxr-xr-x    2 root     root          4096 Apr  4 16:06 sbin
drwxr-xr-x    2 root     root          4096 Apr  4 16:06 srv
dr-xr-xr-x   12 root     root             0 Apr 13 07:04 sys
drwxrwxrwt    1 root     root          4096 Apr 13 07:04 tmp
drwxr-xr-x    1 root     root          4096 Apr 13 07:01 usr
drwxr-xr-x    1 root     root          4096 Apr 13 07:01 var
[+] List contents of public
175-0dcd6aa6c21079d8aa05.js
175-0dcd6aa6c21079d8aa05.js.map
231-4654f5f3ac3fe1e4eaaf.js
231-4654f5f3ac3fe1e4eaaf.js.map
404
404.html
67-4639f193183c83596[22](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:22)6.js
67-4639f193183c83596226.js.map
7112840a-56005de57589deff82e4.js
7112840a-56005de57589deff82e4.js.LICENSE.txt
7112840a-56005de57589deff82e4.js.map
7a4f853b6ed1b1d270733af0138e8a65
app-a1067b4c83e31090b7d9.js
app-a1067b4c83e31090b7d9.js.LICENSE.txt
app-a1067b4c83e31090b7d9.js.map
bugoverdose-logo.png
bugoverdose-open-graph.png
categories
category
chunk-map.json
commons-7dadb1846d649c26545a.js
commons-7dadb1846d649c26545a.js.LICENSE.txt
commons-7dadb1846d649c26545a.js.map
component---src-pages-404-js-9c0125c4edb459993e76.js
component---src-pages-404-js-9c0125c4edb459993e76.js.map
component---src-pages-categories-js-7840dc28e8663edac40d.js
component---src-pages-categories-js-7840dc28e8663edac40d.js.map
component---src-pages-mdx-slug-js-2702498268d1bf8327f2.js
component---src-pages-mdx-slug-js-2702498268d1bf8327f2.js.map
component---src-pages-tags-js-509e3e81871[23](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:23)b8eef4a.js
component---src-pages-tags-js-509e3e8187123b8eef4a.js.map
component---src-templates-category-js-a2d347749b99ab9b64cc.js
component---src-templates-category-js-a2d347749b99ab9b64cc.js.map
component---src-templates-home-js-0f7ea95a8bf2f2e01af6.js
component---src-templates-home-js-0f7ea95a8bf2f2e01af6.js.map
component---src-templates-tag-js-585d7b1dd54cc9e8a5f6.js
component---src-templates-tag-js-585d7b1dd54cc9e8a5f6.js.map
computer-science
development
essay
favicon-32x32.png
framework-d990a81097fc07e0893d.js
framework-d990a81097fc07e0893d.js.LICENSE.txt
framework-d990a81097fc07e0893d.js.map
icons
index.html
manifest.webmanifest
page
page-data
polyfill-4b355b7cb1b252846bfb.js
polyfill-4b355b7cb1b252846bfb.js.map
robots.txt
rss.xml
sitemap
static
styles.e1fc8df16ade7f667a96.css
tag
tags
webpack-runtime-9fa3f628f5389b3d2572.js
webpack-runtime-9fa3f628f5389b3d2572.js.map
webpack.stats.json
[+] Checking if local public exist
[+] Copying contents of source repository folder public to folder  in git repo bugoverdose.github.io
[+] Files that will be pushed
total 5032
drwxr-xr-x   17 1001     121           4096 Apr 13 07:04 .
drwxrwxrwt    1 root     root          4096 Apr 13 07:04 ..
drwxr-xr-x    8 root     root          4096 Apr 13 07:04 .git
-rw-r--r--    1 1001     121            805 Apr 13 07:04 175-0dcd6aa6c21079d8aa05.js
-rw-r--r--    1 1001     121           3391 Apr 13 07:04 175-0dcd6aa6c21079d8aa05.js.map
-rw-r--r--    1 1001     121           3129 Apr 13 07:04 231-4654f5f3ac3fe1e4eaaf.js
-rw-r--r--    1 1001     121           8438 Apr 13 07:04 231-4654f5f3ac3fe1e4eaaf.js.map
drwxr-xr-x    2 1001     121           4096 Apr 13 07:04 404
-rw-r--r--    1 1001     121          11235 Apr 13 07:04 404.html
-rw-r--r--    1 1001     121           1216 Apr 13 07:04 67-4639f193183c83596226.js
-rw-r--r--    1 1001     121           5717 Apr 13 07:04 67-4639f193183c83596226.js.map
-rw-r--r--    1 1001     121         307860 Apr 13 07:04 7112840a-56005de57589deff82e4.js
-rw-r--r--    1 1001     121           7781 Apr 13 07:04 7112840a-56005de57589deff82e4.js.LICENSE.txt
-rw-r--r--    1 1001     121        1346749 Apr 13 07:04 7112840a-56005de57589deff82e4.js.map
drwxr-xr-x    2 1001     121           4096 Apr 13 07:04 7a4f853b6ed1b1d270733af0138e8a65
-rw-r--r--    1 1001     121         146808 Apr 13 07:04 app-a1067b4c83e31090b7d9.js
-rw-r--r--    1 1001     121            487 Apr 13 07:04 app-a1067b4c83e31090b7d9.js.LICENSE.txt
-rw-r--r--    1 1001     121         549553 Apr 13 07:04 app-a1067b4c83e31090b7d9.js.map
-rw-r--r--    1 1001     121          11873 Apr 13 07:04 bugoverdose-logo.png
-rw-r--r--    1 1001     121           [24](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:24)57 Apr 13 07:04 bugoverdose-open-graph.png
drwxr-xr-x    2 1001     121           4096 Apr 13 07:04 categories
drwxr-xr-x    5 1001     121           4096 Apr 13 07:04 category
-rw-r--r--    1 1001     121            774 Apr 13 07:04 chunk-map.json
-rw-r--r--    1 1001     121         188560 Apr 13 07:04 commons-7dadb1846d649c26545a.js
-rw-r--r--    1 1001     121           2423 Apr 13 07:04 commons-7dadb1846d649c26545a.js.LICENSE.txt
-rw-r--r--    1 1001     121         7007[25](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:25) Apr 13 07:04 commons-7dadb1846d649c[26](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:26)545a.js.map
-rw-r--r--    1 1001     121           6331 Apr 13 07:04 component---src-pages-404-js-9c0125c4edb459993e76.js
-rw-r--r--    1 1001     121          15321 Apr 13 07:04 component---src-pages-404-js-9c0125c4edb459993e76.js.map
-rw-r--r--    1 1001     121          15807 Apr 13 07:04 component---src-pages-categories-js-7840dc28e8663edac40d.js
-rw-r--r--    1 1001     121          31672 Apr 13 07:04 component---src-pages-categories-js-7840dc28e8663edac40d.js.map
-rw-r--r--    1 1001     121          10908 Apr 13 07:04 component---src-pages-mdx-slug-js-[27](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:27)02498268d1bf8327f2.js
-rw-r--r--    1 1001     121          31718 Apr 13 07:04 component---src-pages-mdx-slug-js-2702498268d1bf8327f2.js.map
-rw-r--r--    1 1001     121          14629 Apr 13 07:04 component---src-pages-tags-js-509e3e8187123b8eef4a.js
-rw-r--r--    1 1001     121          34729 Apr 13 07:04 component---src-pages-tags-js-509e3e8187123b8eef4a.js.map
-rw-r--r--    1 1001     121          10902 Apr 13 07:04 component---src-templates-category-js-a2d347749b99ab9b64cc.js
-rw-r--r--    1 1001     121          30407 Apr 13 07:04 component---src-templates-category-js-a2d347749b99ab9b64cc.js.map
-rw-r--r--    1 1001     121          12232 Apr 13 07:04 component---src-templates-home-js-0f7ea95a8bf2f2e01af6.js
-rw-r--r--    1 1001     121          31018 Apr 13 07:04 component---src-templates-home-js-0f7ea95a8bf2f2e01af6.js.map
-rw-r--r--    1 1001     121          10966 Apr 13 07:04 component---src-templates-tag-js-585d7b1dd54cc9e8a5f6.js
-rw-r--r--    1 1001     121          30598 Apr 13 07:04 component---src-templates-tag-js-585d7b1dd54cc9e8a5f6.js.map
drwxr-xr-x    6 1001     121           4096 Apr 13 07:04 computer-science
drwxr-xr-x    9 1001     121           4096 Apr 13 07:04 development
drwxr-xr-x    3 1001     121           4096 Apr 13 07:04 essay
-rw-r--r--    1 1001     121            4[28](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:28) Apr 13 07:03 favicon-32x32.png
-rw-r--r--    1 1001     121         1[29](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:29)753 Apr 13 07:04 framework-d990a81097fc07e0893d.js
-rw-r--r--    1 1001     121            736 Apr 13 07:04 framework-d990a81097fc07e0893d.js.LICENSE.txt
-rw-r--r--    1 1001     121         [31](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:31)9864 Apr 13 07:04 framework-d990a81097fc07e0893d.js.map
drwxr-xr-x    2 1001     121           4096 Apr 13 07:03 icons
-rw-r--r--    1 1001     121          11254 Apr 13 07:04 index.html
-rw-r--r--    1 1001     121           1005 Apr 13 07:03 manifest.webmanifest
drwxr-xr-x    3 1001     121           4096 Apr 13 07:04 page
drwxr-xr-x   14 1001     121           4096 Apr 13 07:04 page-data
-rw-r--r--    1 1001     121          85840 Apr 13 07:04 polyfill-4b355b7cb1b252846bfb.js
-rw-r--r--    1 1001     121         205[33](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:33)7 Apr 13 07:04 polyfill-4b[35](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:35)5b7cb1b252846bfb.js.map
-rw-r--r--    1 1001     121            124 Apr 13 07:04 robots.txt
-rw-r--r--    1 1001     121         611841 Apr 13 07:04 rss.xml
drwxr-xr-x    2 1001     121           4096 Apr 13 07:04 sitemap
drwxr-xr-x   52 1001     121           4096 Apr 13 07:04 static
-rw-r--r--    1 1001     121           2659 Apr 13 07:04 styles.e1fc8df16ade7f667a96.css
drwxr-xr-x   27 1001     121           4096 Apr 13 07:04 tag
drwxr-xr-x    2 1001     121           4096 Apr 13 07:04 tags
-rw-r--r--    1 1001     121           4265 Apr 13 07:04 webpack-runtime-9fa3f628f5[38](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:38)9b3d2572.js
-rw-r--r--    1 1001     121          18869 Apr 13 07:04 webpack-runtime-9fa3f628f5389b3d2572.js.map
-rw-r--r--    1 1001     121           [48](https://github.com/bugoverdose/blog/runs/6002905786?check_suite_focus=true#step:7:48)03 Apr 13 07:04 webpack.stats.json
[+] Adding git commit
fatal: unsafe repository ('/tmp/tmp.KPHMGb' is owned by someone else)
To add an exception for this directory, call:

	git config --global --add safe.directory /tmp/tmp.KPHMGb
```

**Comment by cpina** [comment]
Could you try again with `devel` ?

Now it should install git 2.35.2 which is the first version supporting the option that should fix the problem.
(it's now using edge Alpine repository and you should see the git version to be 2.35.2 instead of 2.34.2)

Fingers crossed. If not working I'll try to reproduce on my side, thanks for the patience.

**Comment by a47ae** [comment]
I just tested it and it is still broken for me with `devel`

**Comment by zoha-im** [comment]
The git version updated 
![image](https://user-images.githubusercontent.com/67825158/163125512-ed86179b-cad3-44ec-8167-f5b262e7c42e.png)

But the issue still persists. 
![image](https://user-images.githubusercontent.com/67825158/163126089-0bde5d8f-0e4b-46a8-a68f-f386a3e98720.png)

**Comment by cpina** [comment]
I've made one more attempt in devel (trying to add the specific directory instead of `/`)

Do you mind trying once more? Sorry for the inconvenience!

If it fails again I'll try to setup the exact same problem but I might not have time until later on.

**Comment by a47ae** [comment]
It seems to work now 🎉
Thank you veery much for your quick response :)

**Comment by zoha-im** [comment]
Yes, ran without any issues!🚀 Thank you @cpina for your quick response!

**Comment by bugoverdose** [comment]
It's beautiful. Thanks for the quick fix :)

**Comment by cpina** [comment]
Thanks all for your help! I've pre-released `v1.4.2` and also available in `main`.
